ChangeLog.md

```diff
@@ -23,6 +23,12 @@ Current Trunk
 - dlopen, in conformace with the spec, now checks that one of either RTDL_LAZY
   or RTDL_NOW flags ar set.  Previously, it was possible set nether of these
   without generating an error.
+- The ABI used symbol imports/export in dynamic linking (MAIN_MODULE +
+  SIDE_MODULE) is not that same as the ABI used by llvm and wasm-ld.  That
```

Comment:
This text confuses me. It doesn't describe the actual change, but says something about the ABI. Is that the ABI before this change, or after it?

Reply:
Sorry there was some typos there. 

It should have read:  "The ABI used for symbol imports/export in dynamic linking is *now* the same..".  

Does it make more sense now.  I'm describing the new ABI.